tests/cases/fourslash/completionsDefaultExport.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path="fourslash.ts" />
+
+// @Filename: /a.ts
+////export default function f() {}
+
+// @Filename: /b.ts
+////import * as a from "./a";
+////a./**/;
+
+goTo.marker();
+verify.completionListContains("default", "function f(): void");

```

Comment:
No, the property name is `default`; `function f(): void` is just the display text.